src/components/NihLinkWarning.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'react'
+import { div } from 'react-hyperscript-helpers'
+import { clearNotification, notify } from 'src/components/Notifications'
+import { authStore } from 'src/libs/auth'
+import * as Utils from 'src/libs/utils'
+import * as Profile from 'src/pages/Profile'
+
+
+export const NihLinkWarning = () => {
+  const { nihStatus } = Utils.useAtom(authStore)
+
+  useEffect(() => {
+    const notificationId = 'nih-link-warning'
+
+    if (nihStatus) {
+      const { linkedNihUsername, linkExpireTime } = nihStatus
+      const expireTime = linkExpireTime * 1000
+      const expired = expireTime > Date.now()
+      const expiringSoon = !expired && expireTime < (Date.now() + (1000 * 60 * 60 * 24))
+
+      if (!!linkedNihUsername && (expired || expiringSoon)) {
+        const expirationMessage = expired ? 'has expired' : 'will expire soon'
+        notify('info', div({}, [
+          `Your access to NIH Controlled Access workspaces and data ${expirationMessage} and your access to NIH Controlled Access workspaces will be revoked. `,
```

Comment:
```suggestion
          `Your access to NIH-controlled workspaces and data ${expirationMessage}. To regain access,  `,
```

Reply:
I think "Controlled Access" is an understood term. It might read better if hyphenated ("Controlled-Access") instead of removing the word "Access". However, I'm not positive that hyphenation would be correct. Therefore, I'm going to leave that bit as-is (copied directly from FireCloud). I like the rest of the edits and will incorporate them. Thanks!

Reply:
Sounds good! My goal was to reduce the wordiness and the number of times the notification says "access" :)